src/amber/controller/error.cr

```diff
@@ -13,6 +13,12 @@ module Amber::Controller
 
     def internal_server_error
       response_format("ERROR: #{@ex.inspect_with_backtrace}")
+      # IMPORTANT: #inspect_with_backtrace will fail in some situtions which breaks the tests.
+      # Even if you call @ex.callstack you'll notice that backtrace is nil.
+      # #backtrace? is supposed to be safe but it exceptions anyway.
+      # Please don't remove this without verifying that crystal core has been fixed first.
```

Comment:
Is there an issue on crystal-Lang about this that can be left here in the comment?

Reply:
@robacarp - It is fixed in crystal master.